93 debug view

hhirsch opened the pull request.

SGOrava commented on source/renderer/GameRenderer.cpp:
```diff
@@ -90,11 +90,17 @@ void GameRenderer::renderScene(RenderContext &renderContext) {
   renderEntities(renderContext);
 
   glClear(GL_DEPTH_BUFFER_BIT);
+  if(world.getConfig().isDebugViewEnabled()) {
```
missing space after if
`if (world...`

SGOrava commented:
i think it is better to use "enable" and no "enabled"

hhirsch commented:
@SGOrava The "enable", "enabled" is a good thing to bring up. When I added my option there I saw "enabled" being used there before and when Dorian added his config settings he used "enable".

To me it does not make a big difference.
@SGOrava @ElementW can you each tell me your preference. I will open a new issue for it since it is unrelated to this issue and then use whatever we come up as the preference.

@SGOrava I fixed the format of the if clause. Thank you for pointing it out.

ElementW commented:
@hhirsch IMO all options that are verbs should be in the infinitive form.

hhirsch commented:
@ElementW Then infinitive it shall be. Putting this change in this same PR.

hhirsch commented:
@SGOrava @ElementW Fixed.
